content/tracing/go.md

```diff
@@ -8,15 +8,43 @@ customnav: tracingnav
 
 ### Installation
 
-To begin tracing applications written in Go, first [install and configure the Datadog Agent](/tracing#installing-the-agent).
+To begin tracing applications written in Go you need to:
 
-Next, install the Go Tracer from the Github repository:
+ - [Install and configure the Datadog Agent](/tracing#installing-the-agent).
+
+ -  Install the Go Tracer from the Github repository:
 
 ~~~
 go get "github.com/DataDog/dd-trace-go/tracer"
 ~~~
 
-Finally, import the tracer and instrument your code!
+This Package contains Datadog's tracing client. It is used to trace requests as they flow across web servers, databases and microservices so that developers have visibility into bottlenecks and troublesome requests.
+
+Package tracer has two core objects: Tracers and Spans. Spans represent a chunk of computation time. They have names, durations, timestamps and other metadata. Tracers are used to create hierarchies of spans in a request, buffer and submit them to the server.
+
+The tracing client can perform trace sampling. While the trace agent already samples traces to reduce bandwidth usage, client sampling reduces performance overhead.
+
+  - Import the tracer and instrument your code!
```

Comment:
Since the second bullet point now has a larger body, this third bullet point feels awkward. Let's omit it (the **Example** section shows how to do it, anyway)